pkg/auth/handler/webapp/authflowv2/reset_password.go

```diff
@@ -231,7 +173,3 @@ func (h *AuthflowV2ResetPasswordHandler) serveHTTPAuthflow(w http.ResponseWriter
 		h.Controller.HandleStep(r.Context(), w, r, &handlers)
 	}
 }
-
-func isURLFromAdminAPI(r *http.Request) bool {
-	return r.URL.Query().Get(otp.FromAdminAPIQueryKey) == "true"
```

Comment:
I think we can keep the constant commented, but no longer writes it. So we can grep it later in the codebase and git blame the history if needed.

Reply:
Amended the commit so that we only commented this code, and added link to this pr.